src/controller.ts

```diff
@@ -153,6 +153,10 @@ export default function (
       return paymentService.getPrices();
     });
 
+    fastify.get<{ Body: { email: string; coupon_code: string } }>('/apply-coupon', async (req, rep) => {
+      return paymentService.hasUserAppliedCoupon(req.body.email, req.body.coupon_code);
```

Comment:
Please use camelCase. I know we have some snake case going on, but that is technical debt, we should use camel Case